eppo_core/src/event_ingestion/sdk_key_decoder.rs

```diff
@@ -0,0 +1,61 @@
+use base64::engine::general_purpose::URL_SAFE_NO_PAD;
+use base64::prelude::*;
+
+const PATH: &str = "v0/i";
+
+pub fn decode_event_ingestion_url(sdk_key: &str) -> Option<String> {
```

Comment:
Ooops, I think we clashed. I started working on spawning event ingestion on the background runtime and realized I need some public API that would trigger it, so I started working on initialization and config. I quickly discovered that this decoding part is missing so here it is: #181

Reply:
Cool no worries, sounds like we started working on the same stuff :)